vendor/github.com/containernetworking/plugins/plugins/main/loopback/loopback.go

```diff
@@ -1,72 +0,0 @@
-// Copyright 2016 CNI authors
```

Comment:
Do we want to  remove these files from vendor dir? Removing this plugin from openshift packaging/installation is not sufficient?
I see other plugins under vendor/github.com/containernetworking/plugins/plugins/main which are not directly used by openshift SDN.

Reply:
@pravisankar yeah, those exist there.  we don't build or install them though, so we could either remove them from Godeps or just leave them for now.  No reason to have those sources around if we don't use them I think.